README.md

```diff
@@ -359,11 +359,9 @@ To compress PNGs and SVGs in the current directory and its subdirectories, two c
 
 # Releasing
 
-1. bump version number in `package.json`
-2. `npm install` to update `package-lock.json`
-3. `npm run generate-changelog` to update `CHANGELOG.md`
-4. `npm run release`
-5. try to work out how you can commit the CHANGELOG including the change that includes the CHANGELOG update before you run the release script
+1. Bump the version using `npm version minor` or `npm version major` as appropriate.
```

Comment:
You don't actually want to do this right? Because `npm run release` does this for you!

Reply:
I'm having the script run `npm version patch` by default. So we should still manually run `npm version minor` or `major` as appropriate.